tests/test_export.py

```diff
@@ -0,0 +1,131 @@
+import pytest
+import json
+from jsonschema import validate
+
+from tabgenie.utils.export import *
+
+'''-------------------JSON-SCHEMA-----------------------'''
+# https://python-jsonschema.readthedocs.io/en/latest/
+# need schema for testing tabs exporting into json
+with open('tests/schemes/schema_without_prop.json') as f:
+    schema_without_prop = json.load(f)
+
+with open('tests/schemes/schema_with_prop.json') as f:
+    schema_with_prop = json.load(f)
+'''-----------------------------------------------------'''
+
+
+def test_export_json(prepare_tests):
+    tab, split, len_tab, cls = prepare_tests
+    failed_nums = []
+    for nmb in range(len_tab):
+        try:
+            table = cls.prepare_table(cls.data[split][nmb])
+            for prop in [True, False]:
+                try:
+                    obj_json = json.dumps(table_to_json(table, prop))
+                    if not prop:
+                        schema = schema_without_prop
+                    else:
+                        schema = schema_with_prop
+                    validate(instance=obj_json, schema=schema)
+                except:
+                    failed_nums.append(nmb)
+        except:
+            failed_nums.append(nmb)
+
+    assert not failed_nums, \
+        f'{tab}-{split}\n' \
+        f'Count error tables - {len(failed_nums)}\n' \
+        f'First 20 numbers - {failed_nums[:20]}'
+
+
+def test_export_excel(prepare_tests):
+    tab, split, len_tab, cls = prepare_tests
```

Comment:
what type is `prepare_tests` here? is it really a function object? if not (if it's a tuple), better to rename it to `prepared_test_items` or something like this
(also for other tests)

Reply:
This is a fixture that prepares a dataset and returns a tuple with data for tests.